src/ts/scene/ShopScene.ts

```diff
@@ -36,25 +54,32 @@ export default class ShopScene extends Scene
                 this.scale.startFullscreen();
             }
         })
+
+        this.load.image(ShopScene.KEY.IMAGE.ShopOwner,"assets/images/shopScene/ShopOwner.png");
+        this.load.image(ShopScene.KEY.IMAGE.ShopPaper,"assets/images/shopScene/paper-plain.png");
+        this.load.image(ShopScene.KEY.IMAGE.ShopMyDec,"assets/images/shopScene/MYDEC.png");
+        this.load.image(ShopScene.KEY.IMAGE.ShopExit,"assets/images/shopScene/EXIT.png");
+
+        /** 더미 카드 이미지 */
+        this.load.image(ShopScene.KEY.IMAGE.ShopPotion,"assets/iamges/shopScene/potion.png");
     }
 
     create(): void
     {
         new TopMenu(this, 0, 0);
 
+        const dec : Array<string> = this.shuffledDec();
 
-        const shopproduct = new Shopproduct(this, 0,10);
+        this.add.image(this.cameras.main.width,TopMenu.HEIGHT + 10,ShopScene.KEY.IMAGE.ShopOwner).setOrigin(1, 0).setScale(0.5);
 
-        this.add.image(this.cameras.main.width,TopMenu.HEIGHT + 10,"ShopOwner").setOrigin(1, 0).setScale(0.5);
+        this.add.image(5,40,ShopScene.KEY.IMAGE.ShopPaper).setOrigin(0,0);
 
-        this.add.image(5,40,"ShopPaper").setOrigin(0,0);
+        this.add.image(this.cameras.main.width+70,600,ShopScene.KEY.IMAGE.ShopMyDec).setOrigin(1,0.5).setScale(0.27);
+        this.add.image(this.cameras.main.width+70,700,ShopScene.KEY.IMAGE.ShopExit).setOrigin(1,0.5).setScale(0.27);
 
-        this.add.image(this.cameras.main.width+70,600,"MYDEC").setOrigin(1,0.5).setScale(0.27);
-        this.add.image(this.cameras.main.width+70,700,"EXIT").setOrigin(1,0.5).setScale(0.27);
-
-        const card1 = new ShopCard(this, 0, shopproduct.getBuyableCards().pop(), true).setScale(0.9);
-        const card2 = new ShopCard(this, 0, shopproduct.getBuyableCards().pop(), true).setScale(0.9);
-        const card3 = new ShopCard(this, 0, shopproduct.getBuyableCards().pop(), true).setScale(0.9);
+        const card1 = new ShopCard(this, 0, dec.pop(), true).setScale(0.9);
+        const card2 = new ShopCard(this, 0, dec.pop(), true).setScale(0.9);
+        const card3 = new ShopCard(this, 0, dec.pop(), true).setScale(0.9);
```

Comment:
반복되는 코드는 리스트를 통해 일반화하여 한번에 처리하도록 합시다. 아래 shuffledDec 메소드를 따로 만들 필요 없이 다음과 같이 한번에 처리할 수 있습니다.
```suggestion
        const cardNameArr: Array<string> = Object.keys(this.game.cache.json.get(LoadScene.KEY.DATA.CARD));
        const cardArr: Array<ShopCard> = Shuffle(cardNameArr).slice(0, 3).map(cardName => new ShopCard(this, 0, cardName, true));
```

88번째 줄에 [card1, card2, card3] 배열 대신에 cardArr 배열을 넣어줍시다.